lib/utilities/assets/unknown.js

```diff
@@ -7,5 +7,20 @@ module.exports = Adapter.extend({
     var errorMessage = 'You tried to use an unknown adapter: `' + this.name +
                        '`. Please pass a supported adapter-type.';
     return Promise.reject(new SilentError(errorMessage));
+  },
+  activate: function() {
+    var errorMessage = 'You tried to use an unknown adapter: `' + this.name +
```

Comment:
Thanks for your contribution. Do you think you could refactor the duplicated code out into a private function?

Also some tests would be appreciated too. :+1: